lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
I don't believe so.  If that turns out to be an issue, which I don't think it should, then we can conditionally prefetch based on the input parameters.

Also, why are there 19 sql requests for this one call?

Sent from my iPhone

> On Oct 5, 2015, at 3:38 AM, wajeeha-khalid <notifications@github.com> wrote:
> 
> In lms/djangoapps/teams/tests/test_views.py:
> 
> > @@ -129,7 +129,7 @@ def test_query_counts(self):
> >          team.add_user(self.user)
> >  
> >          # Check the query count on the dashboard again
> > -        with self.assertNumQueries(19):
> > +        with self.assertNumQueries(20):
> If the request does not contain account_privacy field for update; would not pre-fetching preferences a performance compromise?
> 
> —
> Reply to this email directly or view it on GitHub.
>